### PR TITLE
Create christian-weyer.md

### PR DESCRIPTION
**Pull request by ChristianWeyer**

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=416) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=416) before we can accept your contribution.<br/><br/>:x: ChristianWeyer [sign now](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=416)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet-foundation/website?pullRequest=416) it.</sub>

File: input/community/speakers/christian-weyer.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  - JavaScript
+  - Microservices
+  - Security
+  - Serverless
+  - SignalR
+  - Visual Studio
+  - Visual Studio Code
+  - Web Development
+Blog: https://www.thinktecture.com/de/christian-weyer
+Twitter: @christianweyer
```
Comment (daveaglick) [inline input/community/speakers/christian-weyer.md:23]: FYI - it's expecting a full URL here, so https://twitter.com/christianweyer

Comment (ChristianWeyer) [reply input/community/speakers/christian-weyer.md]: Thx!

**Comment by ChristianWeyer** [comment]
Is there still anything on my side to do? Thanks!

**Review by Arhell — APPROVED** [review]

**Pull request merged**